index.js

```diff
@@ -38,16 +39,66 @@ module.exports.pitch = function(request) {
 			compilation.cache = compilation.cache[subCache];
 		}
 	});
+
+	// If the user doesn't include the magic `__assets__` anywhere, there is no
+	// need to produce an asset file.
+	var includeAssets = false
+
+	// I don't know how to register this in webpack to get a hashed filename.
+	// This is my temporary hack to use while I try to find a good solution.
+	var BAD_HASH = Math.random().toString(36).substring(2, 10)

```

Comment:
[`loaderUtils.interpolateName(this, name, options)`](https://github.com/webpack/loader-utils#interpolatename)